proto/buf/alpha/registry/v1alpha1/display.proto

```diff
@@ -41,6 +54,86 @@ service DisplayService {
   rpc ListManageableUserTemplateRoles(ListManageableUserTemplateRolesRequest) returns (ListManageableUserTemplateRolesResponse);
 }
 
+message DisplayOrganizationElementsRequest {
+  // The ID of the organization for which to check
+  // which elements should be displayed.
+  string organization_id = 1;
+}
+
+message DisplayOrganizationElementsResponse {
+  // Display create organization repository element if true.
+  bool create_repository = 1;
+  // Display create organization plugin element if true.
+  bool create_plugin = 2;
+  // Display create organization template element if true.
+  bool create_template = 3;
+  // Display organization settings element if true.
+  bool settings = 4;
+  // Display update organization settings element if true.
+  bool update_settings = 5;
```

Comment:
As I understand it, there is no difference between these two right now, right? Admins can see and change org settings but can't delete the org, right?